src/Uno.UI/UI/Xaml/Application.cs

```diff
@@ -104,6 +104,8 @@ public ApplicationTheme RequestedTheme
 			set
 			{
 				_requestedTheme = value;
+				// Sync with core application's theme
+				CoreApplication.RequestedTheme = value == ApplicationTheme.Dark ? SystemTheme.Dark : SystemTheme.Light;

```

Comment:
How is this related to DateTransferManager?

Reply:
It is used on iOS to support theme selection on `ShareUIOptions`. As it is in Uno.UWP, I can't directly access `Application.RequestedTheme` there, so I needed to replicate that on `CoreApplication`